hapi/hedera-protobufs/services/consensus_create_topic.proto

```diff
@@ -70,4 +71,19 @@ message ConsensusCreateTopicTransactionBody {
      * If specified, there must be an adminKey and the autoRenewAccount must sign this transaction.
      */
     AccountID autoRenewAccount = 7;
+
+    /**
+     * Access control for update/delete of custom fees. Null if there is no key.
+     */
+    Key fee_schedule_key = 8;
+
+    /**
+      * If the transaction contains a signer from this list, no custom fees are applied.
+      */
+    repeated Key free_messages_key_list = 9;
+
+    /**
+      * The custom fee to be assessed during a message submission to this topic
+      */
+    repeated ConsensusCustomFee custom_fees = 10;
```

Comment:
Text should specify behavior of the field and follow correct formatting:
```suggestion
    /**
     * Access control for update/delete of custom fees.
     * <p>
     * If unset, custom fees CANNOT be set for this topic.<br/>
     * If not set when the topic is created, this field CANNOT be set via update.
     */
    Key fee_schedule_key = 8;

    /**
     * A set of custom fee definitions.<br/>
     * These are fees to be assessed for each submit to this topic.
     * <p>
     * Each fee defined in this set SHALL be evaluated for
     * each message submitted to this topic, and the resultant
     * total assessed fees SHALL be charged.<br/>
     * Custom fees defined here SHALL be in addition to the base
     * network and node fees.
     */
    repeated Key free_messages_key_list = 9;

    /**
     * Access control for update/delete of custom fees.
     * <p>
     * If this field is unset, the current custom fees CANNOT be changed.<br/>
     * If this field is set, that `Key` MUST sign any transaction to update
     * the custom fee schedule for this topic.
     */
    repeated ConsensusCustomFee custom_fees = 10;
```
Note: I am guessing a bit as to behavior in this suggestion; please correct any errors above.

Reply:
Fixed